utils/project_utils.py

```diff
@@ -0,0 +1,12 @@
+import os
+from resources.data import PROJECT_NAME
```

Comment:
I like how you can reference this file now and don't have to write code. Very good optimization. We always have something to learn from you!